packages/context/src/binding-key.ts

```diff
@@ -3,7 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {generateUniqueId} from './unique-id';
+import {generateUniqueId} from '.';
```

Comment:
We should try to avoid import local items from the `index` as it may introduce circular `import`.